recipe-client-addon/jar.mn

```diff
@@ -7,6 +7,7 @@
   lib/ (./lib/*)
   data/ (./data/*)
   skin/  (skin/*)
-  vendor/ (./vendor/*)
 % resource shield-recipe-client-content %content/
   content/ (./content/*)
+% resource shield-recipe-client-vendor %vendor/
```

Comment:
It doesn't. `./mach build` fails with an error about `contentaccessible` not being supported for `resource` files, although my build is a few days old. It also breaks if I put it in the chrome manifest we use the the XPI.